### PR TITLE
DCOS-46146 - Upgrade CentOS 7.5 AMIs to latest to update Docker.

### PR DESCRIPTION
**Pull request by jbarrick-mesosphere**
## High-level description (required)

This upgrades the CentOS 7.5 AMIs used by dcos-launch to the latest from dcos-images. The primary motivation for this is to upgrade Docker on the nodes.

## Corresponding tickets (required)

These JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS-46146](https://jira.mesosphere.com/browse/DCOS-46146) networking integration test instability, to be fixed by upgrading Docker to version 17.12.x

## Checklist for all PRs

  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here: N/A tested by integration tests.
  - [x] **Added or updated any relevant documentation**

[Integration tests](https://teamcity.mesosphere.io/project.html?projectId=DcosIo_DcosLaunch&branch_DcosIo_DcosLaunch=%3Cdefault%3E) were run and

  - [ ] AWS Cloudformation Simple (link to job: )
  - [ ] Azure Resource Manager (link to job: )
  - [ ] Onprem-AWS (link to job: )
  - [ ] Onprem-GCE (link to job: )

### CODE REVIEW

**Comment by cprovencher** [comment]
testing against enterprise here: https://teamcity.mesosphere.io/viewLog.html?buildId=1720479&tab=queuedBuildOverviewTab

**Comment by jbarrick-mesosphere** [comment]
OSS:

- https://teamcity.mesosphere.io/viewLog.html?buildId=1720686&buildTypeId=DcOs_Open_Test_IntegrationTest_AwsOnpremWStaticBackendGroup1
- https://teamcity.mesosphere.io/viewLog.html?buildId=1720682&tab=buildResultsDiv&buildTypeId=DcOs_Open_Test_IntegrationTest_AwsOnpremWStaticBackendGroup2
 - https://teamcity.mesosphere.io/viewLog.html?buildId=1720690&tab=buildResultsDiv&buildTypeId=DcOs_Open_Test_IntegrationTest_AwsOnpremWStaticBackendGroup3
 - https://teamcity.mesosphere.io/viewLog.html?buildId=1720678&buildTypeId=DcOs_Open_Test_IntegrationTest_AwsOnpremWStaticBackendGroup4&tab=buildResultsDiv

**Comment by jbarrick-mesosphere** [comment]
EE:

 - https://teamcity.mesosphere.io/viewLog.html?buildId=1720694&tab=buildResultsDiv&buildTypeId=DcOs_Enterprise_Test_Inte_AwsOnpremWStaticBackendAndSecurityStrictGroup1
 - https://teamcity.mesosphere.io/viewLog.html?buildId=1720698&tab=buildResultsDiv&buildTypeId=DcOs_Enterprise_Test_Inte_AwsOnpremWStaticBackendAndSecurityStrictGroup2
 - https://teamcity.mesosphere.io/viewLog.html?buildId=1720702&tab=buildResultsDiv&buildTypeId=DcOs_Enterprise_Test_Inte_AwsOnpremWStaticBackendAndSecurityStrictGroup3
 - https://teamcity.mesosphere.io/viewLog.html?buildId=1720706&tab=buildResultsDiv&buildTypeId=DcOs_Enterprise_Test_Inte_AwsOnpremWStaticBackendAndSecurityStrictGroup4

**Comment by jbarrick-mesosphere** [comment]
All 8 of the tests passed!

**Comment by jbarrick-mesosphere** [comment]
The one test failure there is due to known flaky test that occurs when we run all of the tests in one group (instead of splitting them). I think this is DCOS-48591, but @gauripowale or @cprovencher  would know for sure.

File: dcos_launch/platforms/aws.py [hunk 1]
```diff
@@ -662,21 +662,21 @@ def get_bootstrap_ip(self):
                                                  'us-west-2': 'ami-0116dcbe0583de7ca'},
     'cent-os-7.4-with-docker-selinux-enforcing': CENTOS_74_WITH_DOCKER_SELINUX_ENFORCING,
     # cent-os-7-dcos-prereqs runs centos 7.5. Taken from https://github.com/dcos/dcos-images/blob/master/centos/7.5/aws/DCOS-1.12.1/docker-1.13.1.git8633870/selinux_enabled_enforcing/dcos_images.yaml # noqa
```
Comment (orsenthil) [inline dcos_launch/platforms/aws.py]: @jbarrick-mesosphere  please replace this comment from the actual location of these AMIs.

It is helpful for our reference.

**Review by orsenthil — CHANGES_REQUESTED** [review]
We will need to update the comment too.

File: dcos_launch/platforms/aws.py (hunk 1 above)
Comment (jbarrick-mesosphere) [reply dcos_launch/platforms/aws.py]: Fixed!

**Review by urbanserj — APPROVED** [review]
🚀

**Review by orsenthil — APPROVED** [review]

**Review by cprovencher — APPROVED** [review]

**Pull request merged**